app/assets/stylesheets/lib/layout.scss

```diff
@@ -88,6 +88,7 @@
     // Modifiers
     // .l-paper--games
     &--games {
+      height: 100%;
```

Comment:
Had to do this because height was only as high as my screen, but games scrolled down below container. Maybe a FF/Chrome discrepancy?

Reply:
It's a flex setting that I forgot. I think I'll square it away, as I tend to avoid height: 100% combined with flex.

Reply:
Do you wanna do that on this branch or at a later date?